sdks/python/apache_beam/examples/snippets/transforms/aggregation/combinevalues.py

```diff
@@ -0,0 +1,246 @@
+# coding=utf-8
```

Comment:
There is a lot of repeated code between this PR and GBK snippets PR. Consider merging them maybe?

Reply:
Changed the examples for more unique use cases